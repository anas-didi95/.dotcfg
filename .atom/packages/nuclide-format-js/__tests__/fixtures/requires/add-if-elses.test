if (one) {
  // Nothing.
} else if (two) {
  // Nothing.
} else {
  // Nothing.
}
