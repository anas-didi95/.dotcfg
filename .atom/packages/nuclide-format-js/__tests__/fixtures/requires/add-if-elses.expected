const one = require('one');
const two = require('two');

if (one) {
  // Nothing.
} else if (two) {
  // Nothing.
} else {
  // Nothing.
}
